scripts/generate-routes.js

```diff
@@ -216,6 +216,7 @@ Object.keys(CURRENT_ROUTES).sort().forEach(scope => {
     }, {})
 
     currentEndpoint.url = newEndpoint.path
+    currentEndpoint.responses = newEndpoint.responses
```

Comment:
Do you have any other suggestion? Maybe generating two files, one with them and another one without them?

Reply:
The script generates two JSON files:

1. `lib/routes.json` which is used by the library
2. `scripts/routes-for-api-docs.json` which is used by the API docs only

Eventually the types should be generated from the latter, it has more meta information. Can you give it a try? I know it's lacking some information like aliases, but it would be good to see how far you can get with it and what the blockers are. Eventually we should be able to generate types from octokit/routes directly

Reply:
Ok, I'll give it a try!